front/lib/api/auth_wrappers.ts

```diff
@@ -212,11 +213,22 @@ export function withPublicAPIAuthentication<T, U extends boolean>(
           getRequiredScope(req, opts.requiredScopes)
         );
         if (decoded.isErr()) {
+          const error = decoded.error;
+          if ((error as TokenExpiredError).expiredAt) {
```

Comment:
Why do we need the as? 
Could we do `if.error?.expiredAt` ?

Reply:
error is of type Error, so `if (error?.expiredAt)` does a `Property 'expiredAt' does not exist on type 'Error'`